src/components/utils/equipment-types.js

```diff
@@ -25,50 +25,62 @@ export const EQUIPMENT_INFOS_TYPES = {
 export const EQUIPMENT_TYPES = {
     SUBSTATION: {
         type: 'SUBSTATION',
+        name: 'substations',
```

Comment:
I think that it's not really he same value, the type's value is "**Substation**" but the name's value is "**substations**.
so for displaying those types, it will not be the same than the existing value,; maybe I have misinterpreted, please confirm or deny that.